web-client/src/presenter/computeds/Public/publicTrialSessionHelper.test.ts

```diff
@@ -0,0 +1,476 @@
+import { DateTime } from 'luxon';
+import { TrialSessionInfoDTO } from '@shared/business/dto/trialSessions/TrialSessionInfoDTO';
+import { publicTrialSessionsHelper } from '@web-client/presenter/computeds/Public/publicTrialSessionsHelper';
+import { runCompute } from '@web-client/presenter/test.cerebral';
+
+describe('publicTrialSessionsHelper', () => {
```

Comment:
Minor, but it looks like we alternate between `TrialSession` and `TrialSessions`. Not worth addressing in this PR. (In fact, it's probably better _not_ to address until 10461 goes out too!)